addons/info/src/index.js

```diff
@@ -83,12 +67,25 @@ export function addInfo(storyFn, context, info, _options) {
   );
 }
 
-export const withInfo = (info, _options) =>
-  addonCompose((storyFn, context) => addInfo(storyFn, context, info, _options));
+export const withInfo = textOrOptions => {
```

Comment:
@UsulPro the reason @tmeasday wanted to keep it to a single argument is for this API:

https://gist.github.com/shilman/792dc25550daa9c2bf37238f4ef7a398#options-types

The API described in the gist is obviously just a proposal, but we think having a single argument makes stuff like this simple to implement. If we have two arguments it gets complicated fast.